avatars/models.py

```diff
@@ -46,6 +46,10 @@ class ImputeMethod(Enum):
     fast_knn = "fast_knn"
 
 
+class JobKind(Enum):
+    avatarization = "avatarization"
```

Comment:
Not sure it's real a convention, but I always see Enums in capitalize letters ...
```suggestion
    AVATARIZATION = "avatarization"
```

Reply:
It's the code gen that makes them lowercase, you'll see that's the case for every enum in the file.